src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/CompositionContractMismatchException.cs

```diff
@@ -56,6 +57,16 @@ public CompositionContractMismatchException(string message, Exception innerExcep
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositionContractMismatchException"/> class with serialized data.
+        /// </summary>
+        /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data about the exception being thrown.</param>
+        /// <param name="context">The <see cref="StreamingContext"/>  that contains contextual information about the source or destination.</param>
+        protected CompositionContractMismatchException(SerializationInfo info, StreamingContext context)
+            : base(info, context)
+        {
+        }
+
 #if FEATURE_SERIALIZATION
```

Comment:
Remove FEATURE_SERIALIZATION symbol everywhere.
Retain the content if it is in an exception type. Delete it if it is not in an exception type.